draft-ietf-rtcweb-jsep.xml

```diff
@@ -1720,7 +1720,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <xref target="RFC4568"></xref> MUST NOT be implemented to
             signal SDES SRTP keying information.</t>
 
-            <t>The
```

Comment:
Seemed it needed to go to be consistent with rest of bullet points but I'm fine either way